dex/testing/firo/electrumx.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+#
+# Set up ElectrumX-Firo (electrumX) regtest server for testing.
+# Expects the firo regnet harness to be running and listening for RPC's on
+# harness.sh ALPHA_RPC_PORT="53768"
+#
+# Devs should maybe install gencerts ;-)
+# go install github.com/decred/dcrd/cmd/gencerts@release-v1.7
```

Comment:
Can you state the dependencies. Is it only Python3 and pip? Also I notice below `python3` and `python` commands, I assume that assumes they point to the same binary. Can all be python3?

Reply:
- The complete dependencies are many and in `contrib/requirements` for electrum and in a `setup.py` for electrumX. Since we download the whole repo those will be there.
- When inside the virtual Venv environment `python` and `pip` is correct rather than `python3` `pip3` ... it is possible to run different interpreters inside venv .. maybe even 2.7

Reply:
> You could also add a readme like https://github.com/decred/dcrdex/blob/master/dex/testing/dcr/README.md

Yes for sure. Even the version of python 3 makes a difference for electrum client and the total requirements for electrum and versions of each takes up an entire directory `contrib/requirements` in the downloaded repo and this should be documented. Electrum required module versions are many and very strict; all checked in the client startup. FYI: I tested with python3.10 everything and python3.7 as a check but not exclusively.

For electrumX the requirements and versions are simpler and found in `setup.py` downloaded by the script to `REPO_DIR`